lib/travis/model/job.rb

```diff
@@ -122,13 +122,8 @@ def decrypted_config
     normalize_config(self.config).deep_dup.tap do |config|
       config[:env] = process_env(config[:env]) { |env| decrypt_env(env) } if config[:env]
       config[:global_env] = process_env(config[:global_env]) { |env| decrypt_env(env) } if config[:global_env]
-      if config[:addons]
-        if addons_enabled?
-          config[:addons] = decrypt_addons(config[:addons])
-        else
-          delete_addons(config)
-        end
-      end
+      delete_addons(config) if config[:addons] && !addons_enabled?
+      config[:addons] = decrypt_addons(config[:addons]) if config[:addons]
```

Comment:
@BanzaiMan 
If i'm reading it correctly, `addons_enabled?` is falsey if its a pull request.

If we want to update this check to only decrypt the jwt token on PRs, then we need a new list of decryptable whitelisted plugins?

That seems like the safest solution.